crates/sb_core/net.rs

```diff
@@ -108,7 +108,18 @@ pub async fn op_net_accept(
     if rx.is_none() {
         return Err(bad_resource("unix channel receiver is already used"));
     }
-    let mut rx = rx.unwrap();
+
+    let rx = rx.unwrap();
+    let mut rx = scopeguard::guard(rx, {
+        let state = state.clone();
```

Comment:
This is where `scopeguard` is used.
@laktek

Reply:
Since `scopeguard` is being used by two crates(base, sb_core), so i've bumped it up to a workspace dependency.

Reply:
ah right! missed it.